source/sonata_simulation.rst

```diff
@@ -65,6 +65,17 @@ node_set
 
 A parameter specifying the cells from which node set should be instantiated for the simulation. The absence of that property means that all (non virtual) nodes of all populations are loaded.
 
+compartment_sets_file
+~~~~~~~~~~~~~~~~~~~~~
+
+*Optional*
+
+Path to a ``compartment_sets.json`` file defining named sets of precise compartment targets (e.g. [node_id, section_id, offset]) used for compartment-level reports or inputs.
```

Comment:
instead of `sets of precise compartment targets (e.g. [node_id, section_id, offset]), can you link to the definition so there is no confusion about what the requirements are/